server/src/models/job_posts.model.js

```diff
@@ -44,15 +44,31 @@ const JobPosting = sequelize.define(
         "Weekly",
         "Annually",
         "Commission",
-        "Base & Commission",
+        "Base_Commission", // Changed for MySQL compatibility
```

Comment:
Looking at this, I'm wondering if the job_applications model also needs to be updated here?: https://github.com/uoftblueprint/tces/blob/main/server/src/models/job_applications.model.js#L69 and a couple more places where there is a space between the values?

Reply:
I looked into it, and apparently MySQL doesn't allow ampersands (&) to be used in ENUM types. I think for the Job Applications model, the ENUM type "R & I" would have to be changed to something so that there is no ampersand used. Otherwise, the spaces are okay.

Reply:
Resolved with commit 7f77d22. I changed the ENUM value "Base_Comission" to "Base Comission" and "R & I" to "R and I" for consistency. Hope that helps!